projects/fusion-ui/components/multi-dropdown/v4/multi-dropdown-v4.component.html

```diff
@@ -117,20 +118,21 @@
 
 <!--INNER TEMPLATES-->
 <!--regular option-->
-<ng-template #regularOption let-option="option">
+<ng-template #regularOption let-option="option" let-indentation="indentation">
     <ng-container *ngIf="!option.isGroup; else groupOption">
         <label
-            class="truncate"
-            [id]="option.id"
-            [for]="id + '-' + option.id"
-            [ngClass]="{'has-icon': option.icon}"
-            [fusionTooltip]="dropdownService.optionToString(option, mappingOptions, {dropdownType: 'multi'})"
-        >
+                class="truncate"
+                [id]="option.id"
+                [for]="id + '-' + option.id"
+                [ngClass]="{'has-icon': option.icon}"
+                (click)="$event.stopPropagation()"
```

Comment:
Not so understand why need (click) if it just do $event.stopPropagation()... Is this element clickable?

Reply:
Because it's checking the checkbox and it runs change detection and for some reason triggers scroll to top